### ISSUE TITLE
Konstanz

### ISSUE THREAD

**Issue by AugustQu** [offenesdresden/ParkAPI#191]
I added a branch: Konstanz

The build failed here. Looking into the log-file I see this hint: total lots should be more than free lots
but it looks like this has nothing to do with the Konstanz-code.

Am I right?
Or: what is the problem with the code for Konstanz?

**Comment by jklmnn** [comment]
No, the total lots should be more than free lots are only warnings that do not make the tests fail. The actual problem is the last message
```
ValueError: time data '2019-05-09 13:41' does not match format '%Y-%m-%dT%H:%M:%S'
```
The correct format for this date would be `%Y-%m-%d %H:%M`-

**Comment by AugustQu** [comment]
ups. corrected. format-string is now: "%d.%m.%Y %H:%M"

**Comment by AugustQu** [comment]
can I start the test by myself? In my fork/branch?

**Comment by jklmnn** [comment]
I misinterpreted the error message, I though the time string came from the website. I added a review how to fix the error to #190.

**Comment by jklmnn** [comment]
Fixed with https://github.com/offenesdresden/ParkAPI/commit/d7f39660f523078f32cc5c741a7f744627ab775a.
